scripts/global.js

```diff
@@ -584,10 +584,14 @@ export async function importMasternode() {
     doms.domMnIP.value = '';
     doms.domMnPrivateKey.value = '';
 
-    if (!ip.includes(':')) {
-        address = `${ip}:${cChainParams.current.MASTERNODE_PORT}`;
-    } else {
-        address = ip;
+    if (ip.includes('.')) {
```

Comment:
Perhaps this can be refactored in to a utility function?
Might be useful to re-use (i.e: for a "config export" feature or such, to make sanitising and checking Masternode addresses cleaner).

Reply:
Also added some regexp, it's not 100% perfect, but shoud catch most of the issues